docs/versions.txt

```diff
@@ -15,6 +15,6 @@ JanusGraph.
 | JanusGraph | Cassandra | HBase | Bigtable | Elasticsearch | Solr | TinkerPop | Spark | Scala
 | 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
 | 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z, 1.0.0-preZ | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
-| 0.2.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z, 1.0.0-preZ | 1.z,2.z,5.z,6.z |  5.3-5.5.z, 6.2-6.6.z, 7.z | 3.2.z | 1.6.z | 2.10.z
+| 0.2.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z, 1.3.z | 0.9.z, 1.0.0-preZ | 1.z,2.z,5.z,6.z |  5.3-5.5.z, 6.2-6.6.z, 7.z | 3.2.z | 1.6.z | 2.10.z
```

Comment:
Should we drop 0.98.z and 1.0.z from the matrix? I don't see these anymore on the official mirror http://www-us.apache.org/dist/hbase/

Reply:
@pluradj Would this apply across the chart? For example we have ES 1.x and 2.x even though they're not supported anymore. In my opinion it's okay to advertise compatibility with older versions. It's up to users which versions they choose (or are forced to) use in their environments.

Reply:
Fair enough, maybe we just need a disclaimer. Something like: "Although JanusGraph may be compatible with older and no longer supported versions of its dependencies, users are warned that there are possible risks and security exposures with running software that is no longer supported or updated. Please check with the software providers to understand their supported versions. Users are strongly encouraged to use the latest versions of the software."